src/applications/financial-status-report/sass/financial-status-report.scss

```diff
@@ -352,19 +352,24 @@ $chapters: veteranInformationChapter, householdIncomeChapter,
   &::part(button) {
     &::after,
     &::before {
-      font-family: "Font Awesome 5 Free"; /* Adjust based on your Font Awesome version */
-      font-weight: 900; /* Use 900 for solid icons */
-      font-size: 1rem;
+      width: 1rem;
+      height: 1rem;
       vertical-align: bottom;
+      display: inline-block;
+      background-color: currentColor;
+      mask-position: 0;
+      mask-size: contain;
     }
   }
 }
 
 .va-button-override va-button[text="Back"] {
   &::part(button) {
     &::before {
-      content: "\f100";
+      content: "";
       margin-right: 8px;
```

Comment:
The `margin-right` should be removed now as it adds too much extra spacing:

![image](https://github.com/user-attachments/assets/78129c02-4424-4cdf-9629-565250cb8507)